.gitlab-ci.yml

```diff
@@ -79,6 +79,7 @@ test_nearlib:
     - ./scripts/waitonserver.sh
     - *build_wasm
     - cd nearlib && npm install && npm run build && npm run doc && npm test && cd ..
+    - mkdir new_project && cd new_project && npm install near-shell && node_modules/near-shell/near new_project && npm install && npm run-script build && npm test && cd ..
```

Comment:
`node_modules/near-shell/near` – should definitely be made runnable as just `near` inside of `npm` script

See https://medium.com/netscape/a-guide-to-create-a-nodejs-command-line-package-c2166ad0452e for more details

Note that it also makes sense to use existing project templating tools, AFAIK this one is most popular https://yeoman.io

Reply:
does it have to be a part of this? I want to have the CI running as soon as possible to avoid regressions

Reply:
Resolving cause in case of usage in npm script everything is fine. Issue specific to CI scripts, which probably should be moved out of .yml (can do separate PR).